docs/UserGuide.md

```diff
@@ -245,28 +246,30 @@ Therefore, to avoid unintentionally losing any information of the employee, if y
 
 #### Finding employees by all specified tags: `find-all-tag`
 
-1. Type in the following [command](#glossary) in the [command box](#glossary) `find-all-tag t/TAG [t/MORE_TAGS]...` where `TAG` is the name of the [tags](#glossary) (case-sensitive), `t/TAG` representing at least one tag should be entered to search and `[t/MORE_TAGS]...` representing optional more tags to be entered. Note that only tags containing the whole word are matched.
+1. Type in the following [command](#glossary) in the [command box](#glossary) `find-all-tag t/TAG [t/MORE_TAGS]...` where `TAG` is to be replaced with the name of the [tags](#glossary), and capitalization of the tag names mattered (case-sensitive). At least one tag should be entered for search and more tags can be entered if you want to search for more. 
```

Comment:
Might want to separate out the point about capitalization into a separate sentence for greater readability e.g. The tag names are case-sensitive.